### ISSUE TITLE
Exclude for Operations without OperationId

### ISSUE THREAD

**Issue by marcelkohl** [apideck-libraries/portman#185]
I've noticed that using the `excludeForOperations` on test contracts is not being validated when the specification does not have the `operationId` explicit declared. Looks like the `src/application/TestSuite.ts` (line 176) excludes automatically the conditions without `operationId`. Is it supposed to be like that as the `operationId` is an [optional attribute](https://swagger.io/docs/specification/paths-and-operations/#operationId)? 

I did a sample log on one of my specifications file to exemplify what is happening:
**portman-config.json**
```json
      ...
      {
        "openApiOperation": "*::/v1/*",
        "excludeForOperations": ["GET::/v1/shelter/lists", "GET::/v1/shelter/lists/{listCategory}/{listIdentifier}"],
        "schemaValidation": {
          "enabled": true
        }
      },
      ...
```

**output log**
```
excludeForOperations: [
  'GET::/v1/shelter/lists',
  'GET::/v1/shelter/lists/{listCategory}/{listIdentifier}'
]

operationId IncludeOnTest operation
---
undefined   False         GET::/v1/shelter/scanner/config
listsAll    False         GET::/v1/shelter/lists
getList     False         GET::/v1/shelter/lists/{listCategory}/{listIdentifier}
getListDiff True          GET::/v1/shelter/lists/{listCategory}/{listIdentifier}/diffs
undefined   False         GET::/v1/shelter/another/lists
undefined   False         GET::/v1/shelter/urls/categories
undefined   False         POST::/v1/shelter/urls/animals
undefined   False         GET::/v1/shelter/files/pictures
undefined   False         POST::/v1/shelter/files/documents
undefined   False         POST::/v1/shelter/files/upload/{sha256}
```

**Comment by thim81** [comment]
Hi @marcelkohl 

Your comment correct.
Portman should skip operations that are excludeOperations that match the operationsID OR the path reference.
The current behaviour skips too much, which is not correct.

We will see how we can overcome this.

**Comment by thim81** [comment]
We have created a fix for the bug in the "excludeForOperations" logic.
The fix will be part of the next release of Portman.

**Comment by thim81** [comment]
@marcelkohl 
This issue should be solved since the 1.12.1 release.

I'll close the issue, but in case it is not solved, feel free to reopen or open a new issue.
